server/metric_descriptors_list.go

```diff
@@ -2,12 +2,23 @@ package server
 
 import (
 	"golang.org/x/net/context"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
 	types "k8s.io/cri-api/pkg/apis/runtime/v1"
 )
 
 // ListMetricDescriptors lists all metric descriptors
 func (s *Server) ListMetricDescriptors(ctx context.Context, req *types.ListMetricDescriptorsRequest) (*types.ListMetricDescriptorsResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "method unimplemented")
+	includedKeys := []string{"cpu", "memory", "network"} // You can modify this as needed
```

Comment:
this should be the same list as https://github.com/cri-o/cri-o/pull/7796/files#diff-b9141e04c913bc0a793240448278d523cd5b1bd22cf036b9a0f3bcf1e32113c3R214